taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs

```diff
@@ -164,15 +164,15 @@ private void LoopConditionHelper_CellContentClick(object sender, DataGridViewCel
 
                     var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginLoopCommand).ToList();
```

Comment:
Could likely be reduced to CommandControls.GenerateCommandsandControls

Reply:
Same is the case here that it's just updating variable names/references in this Command File.